### PR TITLE
Fix RadioButtons colouring bug

### PR DESCRIPTION
**Pull request by karenbtlai**
## Description

- Specified transitions back to Normal state as colours were not updating properly when toggling between light and dark themes

## How Has This Been Tested?

- Manual testing

### CODE REVIEW

File: dev/CommonStyles/RadioButton_themeresources.xaml [hunk 1]
```diff
@@ -211,6 +211,24 @@
...
+                                        </ObjectAnimationUsingKeyFrames>
+                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="CheckOuterEllipse" Storyboard.TargetProperty="Fill">
+                                            <DiscreteObjectKeyFrame KeyTime="0:0:0.083" Value="{ThemeResource RadioButtonOuterEllipseCheckedFill}" />
+                                        </ObjectAnimationUsingKeyFrames>
+                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="CheckGlyph" Storyboard.TargetProperty="Fill">
+                                            <DiscreteObjectKeyFrame KeyTime="0:0:0.083" Value="{ThemeResource RadioButtonCheckGlyphFill}" />
+                                        </ObjectAnimationUsingKeyFrames>
+                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="CheckGlyph" Storyboard.TargetProperty="Stroke">
+                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource RadioButtonCheckGlyphStroke}" />
```
Comment (StephenLPeters) [inline dev/CommonStyles/RadioButton_themeresources.xaml:230]: >"0" [](start = 76, length = 3)

intentional?

**Review by StephenLPeters — APPROVED** [review]
:shipit:

File: dev/CommonStyles/RadioButton_themeresources.xaml (hunk 1 above)
Comment (karenbtlai) [reply dev/CommonStyles/RadioButton_themeresources.xaml:230]: Nope, reverted the KeyTime back to 0.

**Pull request merged**